sdk/monitor/azquery/README.md

```diff
@@ -357,6 +337,7 @@ comments.
 [log_analytics_workspace_create]: https://learn.microsoft.com/azure/azure-monitor/logs/quick-create-workspace?tabs=azure-portal
 [time_go]: https://pkg.go.dev/time
 [time_intervals]: https://en.wikipedia.org/wiki/ISO_8601#Time_intervals
+[troubleshooting_guide]: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/monitor/azure-monitor-query/TROUBLESHOOTING.md
```

Comment:
This is a chicken-and-egg situation. The link is invalid since you're adding the file now. You'll have to add this part later. :(

Reply:
fixed in commit [fee0a94](https://github.com/Azure/azure-sdk-for-go/pull/19399/commits/fee0a94bfa771c52e58fed930ff0c0c2e717f004)